packages/expo-image/src/Image.types.ts

```diff
@@ -143,19 +137,14 @@ export interface ImageProps extends ViewProps {
 
   /**
    * Determines whether to cache the image and where: on the disk, in the memory or both.
-   *
    * - `'none'` - Image is not cached at all.
-   *
    * - `'disk'` - Image is queried from the disk cache if exists, otherwise it's downloaded and then stored on the disk.
-   *
    * - `'memory'` - Image is cached in memory. Might be useful when you render a high-resolution picture many times.
-   * Memory cache may be purged very quickly to prevent high memory usage and the risk of out of memory exceptions.
-   *
+   *   Memory cache may be purged very quickly to prevent high memory usage and the risk of out of memory exceptions.
    * - `'memory-disk'` - Image is cached in memory, but with a fallback to the disk cache.
-   *
    * @default 'disk'
    */
-  cachePolicy?: 'none' | 'disk' | 'memory' | 'memory-disk' | /** @hidden */ null;
```

Comment:
Afaik we both agreed to hide `null`s since it's already marked as optional

Reply:
It looks like we have a regression, this seems to work in the past. Going to investigate this in near future.